packages/frontend-2/components/header/NavBar.vue

```diff
@@ -32,6 +32,7 @@
     </div>
     <PopupsSignIn v-if="!activeUser" />
   </nav>
+  <div class="h-14" />
```

Comment:
You shouldn't ever have multiple root elements in a Vue component, it can break Nuxt navigation and cause other weird hard to debug issues